.circleci/config.yml

```diff
@@ -179,14 +179,13 @@ jobs:
           command: |
             git config --global user.email "velox@users.noreply.github.com"
             git config --global user.name "velox"
+            git checkout main
             make -C velox/docs html
-            git fetch origin gh-pages
             git checkout gh-pages
-            mv velox/docs/_build/html/ docs/
+            cp -R velox/docs/_build/html/* docs
```

Comment:
This won't clean up files that are probably deleted? Should the fix be to remove the `docs` directory?

Reply:
@majetideepak If I remove the docs and copy html over, there will be some files missing and the end result web page will be losing some of the styles.